src/raised-button.jsx

```diff
@@ -35,8 +35,8 @@ function getStyles(props, state) {
 
   const amount = (primary || secondary) ? 0.4 : 0.08;
 
-  let backgroundColor = raisedButton.color;
-  let labelColor = raisedButton.textColor;
+  let backgroundColor = props.backgroundColor || raisedButton.color;
```

Comment:
Yes, you're right. This line is missing at ^0.15.0-alpha.1 version and I didn't notice it in latest code.
Sorry for this, I'll close this PR